docker-hub/publish/insights-analytics.md

```diff
@@ -10,17 +10,52 @@ to metrics as both raw data and summary data for a desired time span. You can
 view number of image pulls by tag or by digest, and get breakdowns by
 geolocation, cloud provider, client, and more.
 
+## View the analytics data
+
+Analytics data for your repositories is available for viewing on the Insights
+and analytics dashboard at the following URL:
```

Comment:
Is "Insights and analytics" written in the UI for the dashboard? If so, minor suggestion to make bold.